json_serializable/lib/src/json_key_utils.dart

```diff
@@ -46,7 +46,7 @@ JsonKey _from(FieldElement element, JsonSerializable classAnnotation) {
     );
   }
 
-  Object _getLiteral(DartObject dartObject, Iterable<String> things) {
+  Object _getLiteral(DartObject dartObject, Iterable<String> typeInformation) {
```

Comment:
Add a doc comment and explain what `typeInformation` is

Reply:
ack